src/Couchbase.Lite.Shared/API/Database/DatabaseChangedEventArgs.cs

```diff
@@ -51,6 +51,7 @@ public class DatabaseChangedEventArgs : EventArgs
         /// <summary>
         /// [DEPRECATED]
         /// </summary>
+        [Obsolete("DatabaseChangedEventArgs is deprecated, please use CollectionChangedEventArgs")]
```

Comment:
It won't hurt anything, but this attribute is not as useful for internal properties, since I assume as a developer you know already